hadoop-hdds/rocks-native/src/main/java/org/apache/hadoop/hdds/utils/db/managed/ManagedSSTDumpIterator.java

```diff
@@ -35,21 +36,14 @@
  */
 public class ManagedSSTDumpIterator implements
         Iterator<ManagedSSTDumpIterator.KeyValue>, AutoCloseable {
-  private static final String SST_DUMP_TOOL_CLASS =
-          "org.apache.hadoop.hdds.utils.db.managed.ManagedSSTDumpTool";
   private static final String PATTERN_REGEX =
-          "'([^=>]+)' seq:([0-9]+), type:([0-9]+) => ";
-
+          "'([\\s\\S]+)' seq:([0-9]+), type:([0-9]+)";
```

Comment:
Can you put some comments here with some context/examples of what is relevant to us?